source/terms.txt

```diff
@@ -0,0 +1,169 @@
+Go Terminology Recommendations
+==============================
+
+.. default-domain:: mongodb
+
+
+This is a subset of terms and situations in which we may need to use
+throughout our Go Driver documentation. We should use these when
+appropriate to disambiguate terms, but do not need to use them if there is
+no other reasonable interpretation. Consult the documentation in the
+:ref:`Sources <golang-terms-sources>` for additional guidance.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 25 75
+
+   * - Term
+     - Description / Recommendation
+
+   * - **package**
+     - | **Description:**
+       | Refers to a list of one or more structs, functions, or methods within a file identified by the text ``package <name>`` at the top.
+       | **Recommendation:**
+       | Use monospace and lowercase to refer to a package name.
+       | Example:
+
+       - "Import the ``colors`` package."
+
+       | When disambiguating which package a method/function belongs to, either use the syntax ``<package>.<func name>()`` or describe it as follows: "``<func name>()`` from the ``<package>`` package.
+
+   * - **function**
+     - | **Description:**
+       | Declared in a package using ``func`` and does not specify a receiver. The function in the example below is ``NewColor()``:
+
+       .. code-block:: go
+
+          func NewColor(name string) *Color {}
+
+       | **Recommendation:**
+       | Refer to functions as "function" if you require the non-specific form. Otherwise, use name within monospace with ``()`` appended to the end of the name, and omit parameter information unless describing the signature.
+       | Only prepend the function name with the package that it comes from when disambiguation seems necessary or when listing links to API documentation.
+       | Examples:
+
+       - "Call the ``NewColor()`` function"
+       - "You can call the ``color.Rename()`` or ``person.Rename()`` method."
+
+   * - **method**
+     - | **Description:**
+       | A function declared with a receiver.
+       | **Recommendation:**
+       | Refer to this as a "method" if you require the non-specific form. Otherwise, use the name within monospace with ``()`` appended to the end of the name, and omit parameter information unless describing the signature. Ensure that you include context on what the receiver is.
+       | Only prepend the function name with the package that it comes from when disambiguation seems necessary or when listing links to API documentation.
+       | Example:
+       | Assume the documentation refers to this method:
+
+       .. code-block:: go
+
+          func (a *Color) isPrimary() bool {}
+
+       | "You can call the ``isPrimary()`` method on a ``Color`` pointer receiver type to determine whether the color is red, yellow, or blue as shown below:"
+
+       .. code-block:: go
+
+          // assume assignment c := NewColor("red")
+          c.isPrimary()
+
+   * - **return values/lists**
```

Comment:
Are the term "return values" and "return lists" considered equivalent? 

If I were to use this comment in writing I take it to mean "return value" is the singular form, and "return list" is the plural form. Is that accurate?